sourcemash/frontend/static/js/views/items_view.js

```diff
@@ -55,6 +64,9 @@ Sourcemash.Views.ItemsView = Backbone.View.extend({
         });
 
         this.itemViews = itemCards;
+        if (this.itemViews.length > 0) {
```

Comment:
Makes sure you only update the read status if the collection is nonempty. The problem is that this.collection isn’t always full when rendered gets called, and an empty collection has no notion of read status.